tests/MongoDB.Driver.Tests/Specifications/Runner/MongoClientJsonDrivenTestRunnerBase.cs

```diff
@@ -356,10 +349,29 @@ protected void SetupAndRunTest(JsonDrivenTestCase testCase)
             SetupAndRunTest(testCase.Shared, testCase.Test);
         }
 
-        protected void VerifyCollectionData(IEnumerable<BsonDocument> expectedDocuments, Action<BsonDocument, BsonDocument> prepareExpectedResult)
+        protected virtual void VerifyCollectionData(
+            IEnumerable<BsonDocument> expectedDocuments,
+            string databaseName,
+            string collectionName,
+            ReadConcern readConcern = null)
+        {
+            VerifyCollectionData(expectedDocuments, databaseName, collectionName, readConcern: readConcern, prepareExpectedResult: null);
+        }
+
+        protected void VerifyCollectionData(
+            IEnumerable<BsonDocument> expectedDocuments,
+            string databaseName,
+            string collectionName,
+            ReadConcern readConcern,
+            Action<BsonDocument, BsonDocument> prepareExpectedResult)
         {
-            var database = DriverTestConfiguration.Client.GetDatabase(DatabaseName).WithReadConcern(ReadConcern.Local);
-            var collection = database.GetCollection<BsonDocument>(CollectionName);
+            if (readConcern == null)
+            {
+                readConcern = ReadConcern.Local;
```

Comment:
old opcode update operations don't support local readconcern

Reply:
Can't we just change to `ReadConcern.Default` for all verifications?

Reply:
I wasn't sure about it, but it looks like we can do it :) Done